src/DevBetterWeb.Infrastructure/Handlers/MemberAddressUpdatedHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DevBetterWeb.Core.Events;
+using DevBetterWeb.Core.Interfaces;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+
+namespace DevBetterWeb.Infrastructure.Handlers
+{
+  class MemberAddressUpdatedHandler : IHandle<MemberAddressUpdatedEvent>
+  {
+    public MemberAddressUpdatedHandler(IConfiguration configuration)
+    {
+      _configuration = configuration;
+    }
+
+    public IConfiguration _configuration { get; }
+
+    public async Task Handle(MemberAddressUpdatedEvent addressUpdatedEvent)
+    {
+      var member = addressUpdatedEvent.Member;
+      var addressParts = member.Address?.Split(',');
+
+      if (addressParts is not null)
+      {
+        var cityDetailsAPIResponse = JObject.Parse(await GetMapCoordinates(addressParts[1]));
```

Comment:
Parsing the address should be its own service. If `GetMapCoordinates` needs a particular part of an address, not a full address string, then rename its `address` parameter to something like `cityCountryString` or similar.

Reply:
I've renamed the  parameter, but I wasn't clear what you meant with "Parsing the address should be its own service"?
Thanks